cpp/data/rss_email.template.en

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE HTML>
+<html>
+    <body>
+Dear {user_name},
+
+The following items from your selected RSS feeds are new:</br>
+    <table>
+        <thead>
+            <tr>
+                <th>Item Title</th>
+                <th>Feed</th>
+                <th>Item URL</th>
+            </tr>
+        </thead>
+        <tbody>
+        {LOOP item_titles,feed_urls,feed_names,item_urls}
+            <tr>
+                <td>{item_titles}</td>
+                <td><a href="{feed_urls}">{feed_names></a></td>
```

Comment:
`2021-03-11T14:50:28 SEVERE ./rss_subset_aggregator: in int main(int, char **) --> caught exception: in Template::ProcessEndOfSyntax: error on line 19 expected '}' after variable expansion but found VARIABLE_NAME!`

Reply:
Fixed.